x/programs/cmd/simulator/src/id.rs

```diff
@@ -1,6 +1,6 @@
 use serde::Serialize;
 
-#[derive(Clone, Copy, Debug, PartialEq, Serialize)]
+#[derive(Clone, Copy, Debug, PartialEq, Serialize, Default)]
```

Comment:
🤔, why do you need a default here now but you didn't before?

Reply:
That's because `TestContext` has `program_id` as a field which is of type `Id`.